app/js/sumo_db.js

```diff
@@ -343,6 +343,13 @@
       endpoint += username + '/request_password_reset/';
 
       return request(endpoint, 'GET');
+    },
+
+    get_top_helpers: function() {
+      var endpoint = API_V2_BASE + 'user/weekly-solutions/';
```

Comment:
This endpoint returns the top 10 *users* (not questions). It isn't configurable right now.